modules/tinymce/src/core/test/ts/atomic/html/FilterNodeTest.ts

```diff
@@ -0,0 +1,57 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { Arr, Fun } from '@ephox/katamari';
+import { assert } from 'chai';
+
+import { AstNode } from 'tinymce/core/api/PublicApi';
+import { traverse } from 'tinymce/core/html/FilterNode';
```

Comment:
```suggestion
import { traverse } from 'tinymce/core/html/FilterNode';
import AstNode from 'tinymce/core/html/Node';
```
Just noticed this import is incorrect sorry Lorenzo... we really do need a linter rule for this.

Reply:
I changed but in `import AstNode from 'tinymce/core/api/html/Node';` because `import AstNode from 'tinymce/core/html/Node';` gave me error

Reply:
Ahh sorry about that, I wasn't at the computer so I didn't double check the path.

Reply:
no problem, I just wrote it to be sure that I did not put the wrong path ^^"